dan-smith.js

```diff
@@ -60,12 +60,17 @@ exports.tengwar = {
     "anga-extended": "S",
     "ungwe-extended": "X",
     // punctuation
+    "colon": "ˆ",
+    "apostrophe": "²",
+    "hyphen": ":",
+    "semi-colon": "Â",
+    "parenthesis": "›",
```

Comment:
Is this a different style of parenthesis? Let’s add a test case so we can see the difference.

Reply:
So.... there's no set character for a colon. I'd just use a pause, one or two dots.